docs/com1DFAAlgorithm.rst

```diff
@@ -436,6 +436,17 @@ The mesh values are updated with the particles properties using
 :ref:`particles to mesh interpolation <DFAnumerics:Particles to mesh>` methods.
 This is used to compute flow thickness, flow velocity and pressure fields from the particle properties.
 
+Compute pressure
+~~~~~~~~~~~~~~~~~
+Pressure is computed following:
+
+.. math::
+
+  p = \mathbf{\rho}  \mathbf{u}²,
+
+where :math:`\rho` is the snow density and :math:`\mathbf{u}` is the flow velocity.
```

Comment:
Just an idea: give the name of the code function? I.e. computePressuer in DFAFunctionsCython?

Reply:
added